generator/routing_generator.cpp

```diff
@@ -241,21 +245,12 @@ void BuildCrossRoutingIndex(std::string const & baseDir, std::string const & cou
   LOG(LINFO, ("Cross mwm routing section builder"));
   classificator::Load();
 
-  CountryFile countryFile(countryName);
-  LocalCountryFile localFile(baseDir, countryFile, 0 /* version */);
-  localFile.SyncWithDisk();
-  Index index;
-  auto p = index.Register(localFile);
-  if (p.second != MwmSet::RegResult::Success)
-  {
-    LOG(LCRITICAL, ("MWM file not found"));
-    return;
-  }
+  SingleMwmIndex singleIndex(my::JoinFoldersToPath(baseDir, countryName + DATA_FILE_EXTENSION));
```

Comment:
Please, keep names simple: rename singleIndex to just index.

Reply:
Done